templates/email/default/_email_settings.html

```diff
@@ -89,7 +95,9 @@
 
 only_column_style = "padding: ${ column_padding }px; vertical-align: top; background-color: $primary_column_background_color; color: $primary_column_text_color;"
 primary_column_style = "vertical-align: top; width: 50%; background-color: $primary_column_background_color; color: $primary_column_text_color;"
+primary_column_style_dark_mode = "vertical-align: top; width: 50%; background-color: $primary_column_background_color_dark_mode !important; color: $primary_column_text_color;"
```

Comment:
Feels like this should set a white color as well, no point in setting something that won't work in case it is used anywhere (as I think it might be in the heading)

Reply:
I thought that would be the easier option and I was tempted to go in that direction. My reason to give it the dark background, was from user perspective, it's nice that if I have dark mode enabled, my email is actually is using dark mode, more than us just forcing the email to override the dark mode to look like the light version.
However I understand this is not a massive issue from a UX perspective. 
Let me know what you think. Happy to go with your option.

Reply:
Sorry I wasn't clear - happy with the dark background, but you set a dark text colour here too - I was thinking that should change to white, not the background :)

Reply:
My bad I thought I had created already the color for the text in the first column. 
I just pushed a commit that adds white color in dark mode.
I just tested it and this time I added the DOCTYPE, that was causing the issue on my previews tests. 
![appmail15dark](https://user-images.githubusercontent.com/13790153/157868086-179f98cf-a254-4b62-b0a2-677f59dc7e42.png)

Let me know if there is anything else, thanks again for the feedback.

Reply:
Also I realised that by adding that last fix it also fixes the <h2> element.